File: snippets/fsharp/VS_Snippets_CLR/consolein/FS/consolein.fs
//<snippet1>
open System

let tIn = Console.In
let tOut = Console.Out

tOut.WriteLine "Hola Mundo!"
tOut.Write "What is your name: "
let name = tIn.ReadLine()

tOut.WriteLine $"Buenos Dias, {name}!"

//</snippet1>

File: snippets/csharp/System/Console/In/consolein.cs
//<snippet1>
using System;
using System.IO;

class InTest
{
    public static void Main()
    {

        TextReader tIn = Console.In;
        TextWriter tOut = Console.Out;

        tOut.WriteLine("Hola Mundo!");
        tOut.Write("What is your name: ");
        string name = tIn.ReadLine();

        tOut.WriteLine($"Buenos Dias, {name}!");
    }
}
//</snippet1>
